include/CLUEstering/CLUEstering.hpp

```diff
@@ -339,6 +436,55 @@ namespace clue {
     alpaka::wait(queue);
   }
 
+  template <uint8_t Ndim>
+  template <typename KernelType>
+  void Clusterer<Ndim>::make_clusters_impl(PointsDevice& dev_points,
```

Comment:
can this be called by the other `make_clusters_impl` to avoid code duplication?
(maybe the `alpaka::wait(queue)` should be put in `make_clusters` directly to wait only once at the end at that point)